Cue/app/tabs/library/DeckView.android.js

```diff
@@ -98,28 +98,39 @@ class DeckView extends React.Component {
       onPress: () => { this.props.navigator.pop() }
     }
 
-    const rightItems = [
+    let rightItems = [
       {
         title: 'Filter flagged cards',
         icon: this.state.isFiltering ? CueIcons.filterToggleSelected : CueIcons.filterToggle,
         onPress: () => this.setState({
           isFiltering: !this.state.isFiltering
         })
-      },
-      {
-        title: 'Add card',
-        icon: CueIcons.plus
-      },
-      {
-        title: 'Deck sharing options',
-        icon: CueIcons.share,
-      },
-      {
-        title: 'Edit',
-        icon: CueIcons.edit
-      },
+      }
     ]
 
+    if (this.props.deck.accession === 'private') {
+      rightItems = rightItems.concat([
+        {
+          title: 'Add card',
+          icon: CueIcons.plus
+        },
+        {
+          title: 'Deck sharing options',
```

Comment:
Should we restrict the sharing options to only owned decks? I'd imagine that we could still show the Deck Sharing Options screen, but it would all be read only. (So that people could look up the share codes and stuff.)